core/amber/src/main/scala/edu/uci/ics/texera/web/service/ResultExportService.scala

```diff
@@ -37,6 +37,10 @@ import javax.ws.rs.WebApplicationException
 import javax.ws.rs.core.StreamingOutput
 import java.net.{HttpURLConnection, URL, URLEncoder}
 
+object Constants {
+  val CHUNK_SIZE = 500
```

Comment:
add a comment or modify the variable name to show its describing number of tuples.